### PR TITLE
[9.0][ADD] quality_control_team

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
* [x] :Depends on https://github.com/OCA/manufacture/pull/198

Quality Control Team
=================

Adds Quality Control teams to better manage different flows.

Usage
=====

To use this module, you need to:

#. Go to *Quality Control > Configuration > Quality Control Teams*.

### CODE REVIEW

File: quality_control_team/README.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (MiquelRForgeFlow) [inline quality_control_team/README.rst]: png

File: quality_control_team/README.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
```
Comment (MiquelRForgeFlow) [inline quality_control_team/README.rst]: Icon <https://odoo-community.org/logo.png>`_.

File: quality_control_team/views/qc_team_dashboard.xml [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+                                    <!--<div class="col-xs-4 o_kanban_card_manage_section o_kanban_manage_new">-->
+                                        <!--<div class="o_kanban_card_manage_title">-->
+                                            <!--<span>New</span>-->
+                                        <!--</div>-->
+                                    <!--</div>-->
+                                    <!--<div class="col-xs-4 o_kanban_card_manage_section o_kanban_manage_reports">-->
+                                        <!--<div class="o_kanban_card_manage_title">-->
+                                            <!--<span>Reports</span>-->
+                                        <!--</div>-->
+                                    <!--</div>-->
```
Comment (MiquelRForgeFlow) [inline quality_control_team/views/qc_team_dashboard.xml]: delete commented code

**Review by MiquelRForgeFlow — APPROVED** [review]

File: quality_control_team/views/qc_team_dashboard.xml (hunk 3 above)
Comment (LoisRForgeFlow) [reply quality_control_team/views/qc_team_dashboard.xml]: sure!

**Comment by LoisRForgeFlow** [comment]
@mreficent Thanks a lot! You review has been attended.

**Review by JordiBForgeFlow — APPROVED** [review]

**Pull request merged**